pkg/web/templates/partials/sunrise/sunrise_accept.html

```diff
@@ -0,0 +1,14 @@
+<div class="p-4 my-4 border bg-[#DDFFDD]" hx-target="#sunrise-response" hx-swap="outerHTML">
+  
+  <div class="flex items-baseline gap-x-2">
+    <i class="fa-solid fa-circle-check text-lg text-success"></i>
+    <p class="py-2 text-lg">
+        Success! The secure message transaction has been accepted. A confirmation message will be sent to {originator}. Click the <span class="font-semibold">download</span> button below to save a copy of the transaction data for your records.
+        If you're not yet part of the TRISA network, <a href="https://vaspdirectory.net/" target="_blank" class="underline text-blue-700">consider joining</a> 
+        to streamline future Travel Rule compliance and secure data exchanges.
+    </p>
+  </div>
+  
+  <section id="sunrise-data" class="py-6" hx-get=""></section>
```

Comment:
I've created a separate partial to get the transaction data. If the BE implementation handles this different, the data can be moved to this partial.